src/allmydata/test/test_util.py

```diff
@@ -8,6 +8,7 @@ def foo(): pass # keep the line number constant
 import hashlib
```

Comment:
Outside of this test module, there are no users of `allmydata.util.cachedir`.  Perhaps deleting it is a better option?

Reply:
ooo, nice one. deleted :)